### DISCUSSION TITLE
Inside build updated

### DISCUSSION THREAD

**Discussion by agentwalker** [microsoft/azuredatastudio#25645] [General]
Hi, just a quick question. 
I'm currently running the insider build version 1.49.0.
Usually there's daily updates but for some reason it's saying the last update was 2 weeks ago. Is this right and no updates have be pushed out? Or is the update getting blocked somewhere? 

Cheers

**Comment by Charles-Gagnon** [comment]
This is expected behavior - new releases are only made when there are code changes to publish.
